ssn/config.js

```diff
@@ -111,10 +111,24 @@ var respecConfig = {
       authors: ["Mark Schildhauer", "Matthew B. Jones", "Shawn Bowers", "Joshua Madin", "Sergeui Krivov", "Deana Pennington", "Ferdinando Villa", "Benjamin Leinfelder", "Christopher Jones", "Margaret O'Brien"],
       title: "OBOE: the Extensible Observation Ontology, version 1.1"
     },
-    "SAREF": {
+    "OMS": {
```

Comment:
OMS is in SpecRef so does not need an entry in the local bibliography
https://www.specref.org/?q=oms

Reply:
Fixed